exporter/zipkinexporter/factory.go

```diff
@@ -51,21 +51,19 @@ func (f *Factory) CreateDefaultConfig() configmodels.Exporter {
 
 // CreateTraceExporter creates a trace exporter based on this config.
 func (f *Factory) CreateTraceExporter(logger *zap.Logger, config configmodels.Exporter) (exporter.TraceExporter, error) {
-	cfg := config.(*Config)
+	zc := config.(*Config)
 
-	if cfg.URL == "" {
+	if zc.URL == "" {
 		// TODO https://github.com/open-telemetry/opentelemetry-collector/issues/215
 		return nil, errors.New("exporter config requires a non-empty 'url'")
 	}
+
 	// <missing service name> is used if the zipkin span is not carrying the name of the service, which shouldn't happen
 	// in normal circumstances. It happens only due to (bad) conversions between formats. The current value is a
 	// clear indication that somehow the name of the service was lost in translation.
-	ze, err := newZipkinExporter(cfg.URL, "<missing service name>", 0, cfg.Format)
-	if err != nil {
-		return nil, err
-	}
+	zc.ServiceName = "<missing service name>"
```

Comment:
This line is not needed anymore, otherwise it will ignore what was set on the config. Please move the comments above (lines 61 to 63) to the declaration of `defaultServiceName` on file `exporter/zipkinexporter/exporter.go`